pom.xml

```diff
@@ -109,6 +109,7 @@
                     <!-- TODO: set project ID. -->
                     <deploy.projectId>denniswillie-step-2020</deploy.projectId>
                     <deploy.version>1</deploy.version>
+                    <port>8181</port>
```

Comment:
Seems like this change is unrelated. Can we leave it out?

Reply:
Sure. I removed it.